src/Microsoft.Identity.Web.Certificate/SignedAssertionFilePathCredentialsLoader.cs

```diff
@@ -36,13 +36,14 @@ public async Task LoadIfNeededAsync(CredentialDescription credentialDescription,
                 {
                     // Given that managed identity can be not available locally, we need to try to get a
                     // signed assertion, and if it fails, move to the next credentials
-                    _= await signedAssertion!.GetSignedAssertionAsync(null);
+                    _ = await signedAssertion!.GetSignedAssertionAsync(null);
                     credentialDescription.CachedValue = signedAssertion;
                 }
                 catch (Exception)
                 {
                     credentialDescription.Skip = true;
-                }
+                    throw;
```

Comment:
This is part of the fix, because SignedAssertionFilePath loader would otherwise fail silently and lead to no credentail being used in MSAL. All loaders should throw.